ui/src/pages/deployment-details/deployment-details-form/section-source-images/section-source-images.tsx

```diff
@@ -71,26 +68,7 @@ export const SectionSourceImages = ({
             lastUpdated={lastUpdated}
           />
         </FormRow>
-        {status === Status.Connected ? (
-          <>
-            <FormRow>
-              <InputValue
-                label={translate(STRING.FIELD_LABEL_CAPTURES)}
-                value={deployment.numImages}
-              />
-              <InputValue
-                label={translate(STRING.FIELD_LABEL_EXAMPLE_CAPTURES)}
```

Comment:
I'm thinking we should remove this count. The "Example captures" become regular captures after you upload them and I don't think we should expose info about this proxy model.

<img width="606" alt="image" src="https://github.com/RolnickLab/ami-platform/assets/158175/5ded5bf9-71c5-484e-9cf9-deb4257cd110">